tests/ChildProcess/ChildProcessTest.php

```diff
@@ -2,10 +2,21 @@
 
 use Illuminate\Http\Client\Request;
 use Illuminate\Support\Facades\Http;
+use Mockery;
+use Native\Laravel\ChildProcess as ChildProcessImplement;
+use Native\Laravel\Client\Client;
 use Native\Laravel\Facades\ChildProcess;
 
 beforeEach(function () {
     Http::fake();
+
+    $mock = Mockery::mock(ChildProcessImplement::class, [resolve(Client::class)])
+        ->makePartial()
+        ->shouldAllowMockingProtectedMethods();
+
+    $this->instance(ChildProcessImplement::class, $mock->allows([
+        'fromRuntimeProcess' => $mock,
+    ]));
```

Comment:
The tests failed after the branch update from upstream. I added this partial test double here to address this. 

Do you think this is okay @simonhamp ? It's not the most beautiful solution.

Reply:
I think this is fine for now. What it highlights is that the `Http::fake()` needs to provide more useful 'responses' instead of simply stopping the requests going out

But that's something we can fix later